Source/JavaScriptCore/runtime/GenericTypedArrayViewInlines.h

```diff
@@ -86,6 +86,13 @@ RefPtr<GenericTypedArrayView<Adaptor>> GenericTypedArrayView<Adaptor>::tryCreate
     return result;
 }
 
+template<typename Adaptor>
+RefPtr<GenericTypedArrayView<Adaptor>> GenericTypedArrayView<Adaptor>::tryCreate(Ref<ArrayBuffer>&& buffer)
+{
+    auto length = buffer->byteLength();
+    return adoptRef(new GenericTypedArrayView(WTFMove(buffer), 0, length));
```

Comment:
Could and should add a `*` before `new`, since `new` doesn’t ever return `nullptr` and that’s what will make `adoptRef` return a `Ref`.

Reply:
Currently all the `create()` methods follow this pattern:
```
template<typename Adaptor>
Ref<GenericTypedArrayView<Adaptor>> GenericTypedArrayView<Adaptor>::create(
    RefPtr<ArrayBuffer>&& buffer, size_t byteOffset, std::optional<size_t> length)
{
    auto result = tryCreate(WTFMove(buffer), byteOffset, length);
    RELEASE_ASSERT(result);
    return result.releaseNonNull();
}
```
That is what made me a little less sure about adding this one. But maybe it's okay as there's currently none that takes a `Ref<ArrayBuffer>` and so they all need to allocate. And this only allocates a wrapper.

Reply:
The number of compiler-required explicit `RefPtr { }` wrappers for this change is somewhat unfortunate, but it did reveal one redundant null check.